.github/workflows/test.yaml

```
@@ -0,0 +1,48 @@
+          POSTGRES_PASSWORD: testpassword
+          POSTGRES_DB: mojopoker_test
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)